app/helpers/and.js

```diff
@@ -0,0 +1,4 @@
+import Ember from 'ember';
+import { andHelper } from 'ember-truth-helpers/helpers/and';
+
+export default Ember.Helper.helper(andHelper);
```

Comment:
06e0041d definitely prevents the initializer from running in an application, however it does not prevent the tests in this repo from running `registerHelper` manually (see [here](https://github.com/jmurphyau/ember-truth-helpers/blob/master/tests/unit/helpers/and-test.js#L10) for example).  This means that the tests here are still using `registerHelper` and are unlikely to pass or work on Ember 2.0.0 (current beta cycle) once `Ember.HTMLBars._registerHelper` is removed (which will be happening within the next day or two).

If you move that guard into `registerHelper` instead of in the initializer that would more accurately reflect reality in the tests, but it will likely fail the tests because they are not setup with a resolver.

We need to restructure the tests to use `moduleForComponent` (from `ember-qunit`) like [this example](https://github.com/aptible/dashboard.aptible.com/blob/master/tests/integration/helpers/eq-test.js) for the resolver stuff to be fixed. We should likely refactor the tests in a separate PR, then rebase this one on top of that once landed.  I can tackle updating the tests later today/tomorrow unless you have a chance to do it earlier...